processing/src/main/java/io/druid/segment/DimensionHandlerUtils.java

```diff
@@ -242,7 +256,7 @@ public static Long convertObjectToLong(Object valObj)
   public static Float convertObjectToFloat(Object valObj)
```

Comment:
This method doesn't match `convertObjectToDouble()` in null handling of the result of tryParse(). Also, please locate those methods next to each other.